app/src/main/java/org/oppia/android/app/translation/AppLanguageWatcherMixin.kt

```diff
@@ -36,8 +36,10 @@ class AppLanguageWatcherMixin @Inject constructor(
    * initialized if previous bootstrapping was lost (e.g. due to process death), so it must be
    * called before interacting with the locale handler to avoid inadvertent crashes in such
    * situations.
+   *
+   * @param [Boolean] if shouldOnlyUseSystemLanguage or also user selected language.
```

Comment:
```suggestion
   * @param shouldOnlyUseSystemLanguage whether only the system language should be used
```

A rewording, but also please note the formatting here. This is how a parameter should be documented, and this applies elsewhere in your PR (so please make fixes accordingly).

Reply:
thanks updated.